kernel/common/enumerable.rb

```diff
@@ -314,6 +314,28 @@ def grep(pattern)
     ary
   end
 
+  def grep_v(pattern)
+    ary = []
+
+    if block_given?
```

Comment:
I kinda followed `grep` implementation, is it OK if I changed it there too in this PR?